src/support.sch

```diff
@@ -136,6 +162,34 @@
         </rule>
     </pattern>
 
+    <pattern id="fig">
+        <rule context="fig">
+            <assert test="
+                graphic[
+                    (@mimetype='image' and @mime-subtype='jpeg')
+                    or
+                    ('.jpg'=translate(substring(@xlink:href, string-length(@xlink:href)-3), 'ABCDEFGHIJKLMNOPQRSTUVWXYZ', 'abcdefghijklmnopqrstuvwxyz'))
+                    or
+                    ('.jpeg'=translate(substring(@xlink:href, string-length(@xlink:href)-4), 'ABCDEFGHIJKLMNOPQRSTUVWXYZ', 'abcdefghijklmnopqrstuvwxyz'))
```

Comment:
This got me thinking, maybe we should require `@mimetype` and `@mime-subtype`, and rely on an ingestion step to infer it from file extensions?